misk-redis/src/main/kotlin/misk/redis/FakeRedis.kt

```diff
@@ -0,0 +1,52 @@
+package misk.redis
+
+import misk.time.FakeClock
+import java.time.Instant
+import javax.inject.Inject
+
+class FakeRedis : Redis {
+  @Inject lateinit var clock: FakeClock
+
+  // The value type stored in our key-value store
+  private data class Value (
+    val value: String,
+    val expiryInstant: Instant = Instant.MAX
```

Comment:
is this a typical default for the real thing? you typically want your fakes to behave like the real thing

Reply:
actually this is a private class, so probably doesn't really matter. nit: I would just remove the default since it's easy to set it explicitly

Reply:
Yeah it's only used by `set` and `setex` which set the expiry appropriately, I'll remove the default